### ISSUE TITLE
Ctrl+Alt+Delete

### ISSUE THREAD

**Issue by rotordave** [Flowm/etherkey#2]
Hi,

I have Etherkey working on an RPi 4B (using minicom) but am unable to send Ctrl+Alt+Delete to a Windows 10 PC. I tried using 
`Send ^!{Delete}`
and
`Send ^!{Del}`
and other combinations, but this did not work. Individually the modifier keys work but not together with {Del}. Can you offer any advice?

By the way, I note that on recent Raspian releases I need to use /dev/serial0 rather than /dev/ttyAMA0 - perhaps the documentation can be updated. Speaking of which, what is the best way to Send a command like this from a command line (for me this is the primary use case)? Should I just pipe it to /dev/serial0 ?

**Comment by nummer30** [comment]
Hi,

I don't know any reason why Ctrl+Alt+Delete shouldn't or wouldn't work. But it's been "a few" years since I've done anything with Etherkey.
I'll take a look at your issue, but it will take a few days, probably the coming weekend.

I'll update the Readme to include /dev/serial0, thanks for bringing this to our attention.

For sending commands directly from command line, the python tool **client/teensy-usb.py** is might be what you are looking for.

**Comment by rotordave** [comment]
Thanks for the tip, I was able to use the teensy-usb.py script - though with the same result.

FYI, this is the output I get when sending `^!{Del}`:
```
Send ^!{Del}
        Command: Send            -> 7c6b0019
        Cmd-ext: Del             -> b874048
        Keyname: Del             -> b874048
        SendKey:  61516 = 1111000001001100 | mod:      0 = 00000000
```
If I have notepad open on the target PC I can see the {Del} is effected - the keystroke deletes a character as if a singular Delete keystroke was sent and the Control, Alt keystrokes were not.

**Comment by nummer30** [comment]
Hi,

it was indeed not possible to send any bracketed keyname like `{Delete}` together with modifiers. For example `Send ^c` resulted in the correct Ctrl+C being sent, but `Send ^{c}` only sent a regular character.

I have fixed this in commit cbcbe3773622f34b5598e1663f5598aba5fe631b.

Please let me know if this works for you, so I can close this issue. Thanks.

**Comment by rotordave** [comment]
Works perfectly, thanks nummer30!

(For bonus points, I'm typing this with Etherkey having just unlocked my W10 PC.)
